src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.cs

```diff
@@ -318,8 +318,8 @@ private static bool TryGetApiData(AnalyzerOptions analyzerOptions, Compilation c
                 return false;
             }
 
-            shippedData = ReadApiData(shippedText.Path, shippedText.GetText(cancellationToken), isShippedApi: true);
-            unshippedData = ReadApiData(unshippedText.Path, unshippedText.GetText(cancellationToken), isShippedApi: false);
+            shippedData = ReadApiData(shippedText.Value.path, shippedText.Value.text, isShippedApi: true);
+            unshippedData = ReadApiData(unshippedText.Value.path, unshippedText.Value.text, isShippedApi: false);
```

Comment:
❔ Is this better than just updating `ReadApiData` to check the argument for null?

Reply:
The reason I went with this approach was that I wanted to treat the file being configured but not available on disk the same as if it was unconfigured (IE, it would respect silentlyBailOutOnMissingApiFiles if they were both missing, but give you PublicApiFileMissing if it was just 1).  So TryGetApiText needs to return false if shippedText.GetText(...) is going to return null.  At that point I just figured there was no reason to call it twice, so I just returned the path and content.